### PR TITLE
[Docs] Build Chinese docs

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add Chinese docs. Documentation can be visited at https://mmcv-dev.readthedocs.io/zh/latest/index.html

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -1 +0,0 @@
-../README.md
```
Comment (ZwwWayne) [inline docs/readme.md:1]: Should not be deleted

**Comment by ZwwWayne** [comment]
The image files are too large. Should reuse those in the docs/_static

**Comment by zhouzaida** [comment]
> The image files are too large. Should reuse those in the docs/_static

got it

**Comment by zhouzaida** [comment]
Documentation can be visited at https://mmcv-dev.readthedocs.io/zh/latest/io.html

File: docs_zh_CN/conf.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+version_file = '../mmcv/version.py'
+with open(version_file, 'r') as f:
+    exec(compile(f.read(), version_file, 'exec'))
+__version__ = locals()['__version__']
+
+# -- Project information -----------------------------------------------------
+
+project = 'mmcv'
+copyright = '2018-2019, Kai Chen'
```
Comment (ZwwWayne) [inline docs_zh_CN/conf.py:30]: How about changing it to OpenMMLab authors. see examples in other repo

Comment (hellock) [reply docs_zh_CN/conf.py:30]: also in docs/conf.py

Comment (zhouzaida) [reply docs_zh_CN/conf.py:30]: got it

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**